app/src/main/res/layout-v21/seekbar_pref.xml

```diff
@@ -51,6 +51,28 @@
         />
   </LinearLayout>
 
```

Comment:
Adding these 2 text views under the seekbar, one for arabic and one for the translated language.
They are hidden by default in SeekBarPreference.java and the overriding classes opt in to show them and do something with them (i.e apply text size or brightness on them).